wandb/history.py

```diff
@@ -117,9 +133,8 @@ def torch(self):
 
     def _index(self, row):
         """Internal row adding method that updates step, and keys"""
-        if self._process == "wandb":
-            self.row = row
-            self.rows.append(row)
+        self.row = row
+        self.rows.append(row)
```

Comment:
Maybe a TODO here saying we don't want to bloat the user process memory with the history?